src/components/molecules/ZopaFooter/ZopaFooter.tsx

```diff
@@ -20,18 +20,22 @@ export interface IFooterProps extends HTMLAttributes<HTMLDivElement> {
    * This is useful for pages where by design requires the user to focus on one task.
    */
   legalOnly?: boolean;
+  /**
+   * This sets the base url for all links in the footer
+   */
+  baseUrl?: string;
 }
 
-const ZopaFooter: React.FC<IFooterProps> = ({ legalOnly, ...rest }) => (
+const ZopaFooter: React.FC<IFooterProps> = ({ legalOnly, baseUrl = 'https://www.zopa.com', ...rest }) => (
```

Comment:
I see that we use default props below so it's probably better to do it one way or the other. I know it's been there before but I think you can remove `FC` from here too. 

`const ZopaFooter = ({ legalOnly = false, baseUrl = 'https://www.zopa.com', ...rest }) => (...)`